src/packages/documents/document-types/workspace/views/design/document-type-workspace-view-edit-property.element.ts

```diff
@@ -124,21 +187,7 @@ export class UmbDocumentTypeWorkspacePropertyElement extends UmbLitElement {
 							.value=${this.property.name}
 							@input=${this.#onNameChange}></uui-input>
 						<!-- TODO: should use UUI-LOCK-INPUT, but that does not fire an event when its locked/unlocked -->
```

Comment:
This comment is related to the uui-input so I would say it should be moved into the render method.